tests/example.js

```
@@ -0,0 +1,8 @@
+const unUsedVar = 'I am un-used'
```

ESLint

[error] 'unUsedVar' is assigned a value but never used.

(no-unused-vars)